Loop/Managers/LoopDataManager.swift

```diff
@@ -795,7 +736,7 @@ extension LoopDataManager {
         // Fetch glucose effects as far back as we want to make retroactive analysis
         var latestGlucoseDate: Date?
         updateGroup.enter()
-        glucoseStore.getCachedGlucoseSamples(start: Date(timeIntervalSinceNow: -settings.inputDataRecencyInterval)) { (values) in
+        glucoseStore.getCachedGlucoseSamples(start: Date(timeIntervalSinceNow: -settings.inputDataRecencyInterval), end: nil) { (values) in
```

Comment:
I *think* you are going to need to do a search for all `Date()` and `timeIntervalSinceNow` and replace with appropriate versions using `now()`.